imagetest/testworkflow_test.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 func TestAddStartStep(t *testing.T) {
-	twf, err := NewTestWorkflow("name", "image")
+	twf, err := NewTestWorkflow("name", "project", "zone", "image")
```

Comment:
project and zone are not the responsibility of the test author; they should be added in finalize function